test/indexer/test_hca_indexer.py

```diff
@@ -860,6 +861,54 @@ def test_metadata_field_exclusion(self):
                                      })
         self.assertEqual(1, hits["hits"]["total"])
 
+    def test_variable_metadata_format_indexable(self):
+        """
+        Index different formats for a single field in metadata and ensure dynamic mapper in
+        elasticsearch does not set a field type for incoming data.
+        """
+        multiple_cell_line_bundle = ('e0ae8cfa-2b51-4419-9cde-34df44c6458a', '2018-12-05T230917.591044Z')
+        manifest_multiple_cell_lines, metadata_multiple_cell_lines = self._load_canned_bundle(multiple_cell_line_bundle)
+
+        manifest_single_cell_line, metadata_single_cell_line = self._load_canned_bundle(self.new_bundle)
+
+        # Patch bundles to obtain desired response from Elasticsearch dynamic type setter
+        entity = 'cell_line_0.json'
+
+        metadata_single_cell_line[entity] = deepcopy(metadata_multiple_cell_lines[entity])
+        metadata_single_cell_line[entity]['date_established'] = '2014-10-24'
+        metadata_single_cell_line[entity]['provenance'] = {'document_id': 'e7e0f358-a681-4412-8888-318234f90ca9',
```

Comment:
See wrapping of literals in https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#1code-style
```suggestion
        metadata_single_cell_line[entity]['provenance'] = {
            'document_id': 'e7e0f358-a681-4412-8888-318234f90ca9',
```